src/main/webapp/services/AssignUserService.js

```diff
@@ -0,0 +1,150 @@
+import axios from 'axios';
+import { getIntegrationBaseUrl } from 'symphony-integration-commons';
+import BaseService from './BaseService';
+
+const assignDialog = require('../templates/assignDialog.hbs');
+const errorDialog = require('../templates/errorDialog.hbs');
+const unexpectedErrorDialog = require('../templates/unexpectedErrorDialog.hbs');
+const forbiddenDialog = require('../templates/forbiddenDialog.hbs');
+const notFoundDialog = require('../templates/issueNotFoundDialog.hbs');
+const successDialog = require('../templates/successDialog.hbs');
+
+export default class AssignUserService extends BaseService {
+  constructor(serviceName) {
+    super(serviceName);
+    this.baseUrl = getIntegrationBaseUrl();
+    this.selectedUser = {};
+  }
+
+  successDialog(issueKey) {
+    this.closeDialog('assignIssue');
+
+    const template = successDialog({
+      issueKey,
+      prettyName: this.selectedUser.prettyName,
+    });
+
+    this.openDialog('userAssigned', template, {});
+  }
+
+  openAssignDialog(data, serviceName) {
+    const dialogsService = SYMPHONY.services.subscribe('dialogs');
+
+    const template = assignDialog({
+      url: data.entity.issue.url,
+      key: data.entity.issue.key,
+      name: data.entity.issue.assignee.displayName,
+    });
+
+    const userData = {
+      user: {
+        service: serviceName,
+        crossPod: 'NONE',
+      },
+      assignIssue: {
+        service: serviceName,
+        label: 'OK',
+        data: {
+          entity: data.entity,
+          type: 'assignIssue',
+        },
+      },
+      closeAssignDialog: {
+        service: serviceName,
+        label: 'Cancel',
+        data: {
+          entity: data.entity,
+          type: 'closeAssignDialog',
+        },
+      },
+    };
+
+    dialogsService.show('assignIssue', serviceName, template, userData, {});
+  }
+
+  searchAssignableUser(url, issueKey) {
```

Comment:
The axios calls can be put in a separated file, as FE-Commons. This way the urls are kept in one place. 
Wdyt?